src/apps/backend/modules/shared-task/internal/store/shared-task-db.ts

```diff
@@ -0,0 +1,31 @@
+import { Schema, Types } from 'mongoose';
+
+export interface SharedTaskDB {
+  _id: Types.ObjectId;
+  task: Types.ObjectId;
+  account: Types.ObjectId;
+}
+
+export const SharedTaskDbSchema: Schema = new Schema<SharedTaskDB>(
+  {
+    task: {
+      type: Schema.Types.ObjectId,
+      ref: 'Task',
+      index: true,
+      required: true,
+    },
+    account: {
+      type: Schema.Types.ObjectId,
+      ref: 'Account',
+      index: true,
+      required: true,
+    },
```

Comment:
we can add an active field here so that we can support the unshare of tasks functionality in the future.